spec/requests/api/v1/transactions/create_transfer_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+describe 'POST /api/organizations/#/transactions/transfer' do
+  let(:path) { "/api/organizations/#{organization.id}/transactions/transfer" }
+
+  let(:from_bank_account) { create :bank_account, organization: organization }
+  let(:to_bank_account) { create :bank_account, organization: organization }
+  let(:amount) { Money.new(10000, from_bank_account.currency) }
+  let(:comission) { Money.new(1000, to_bank_account.currency) }
+  let(:exchange_rate) { 2.5 }
+  let(:comment) { 'Test comment' }
+
+  let!(:owner) { create :user }
+  let!(:user) { create :user }
+  let!(:organization) { create :organization, owner: owner, with_user: user }
+  let(:params) {
+    {
+      transfer: {
+        amount: amount,
+        comission: comission,
+        bank_account_id: from_bank_account.id,
+        reference_id: to_bank_account.id,
+        exchange_rate: exchange_rate,
+        comment: comment,
+        date: Time.current
+      }
+    }
+  }
+
+  context 'unauthenticated' do
+    it { post(path) && expect(response).to(be_unauthorized) }
+  end
+
+  context 'authenticated as owner' do
```

Comment:
@yunixon you don't need to test both 'owner' and 'user' because we don't have specific logic for owner. testing as normal user will be enough